packages/interface/src/api/funding/endpoint/apiFnd001.ts

```diff
@@ -0,0 +1,421 @@
+    }),
+  ),
+  fixtureSoftwareEvidence: z.coerce.string().optional(),
+  fixtureSoftwareEvidenceFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  numberOfFixture: z.coerce.number().int().min(1).optional(),
+  priceOfFixture: z.coerce.number().int().min(1).optional(),
+
+  isTransportation: z.coerce.boolean(),
+  transportationEnumId: z.nativeEnum(TransportationEnum).optional(),
+  origin: z.coerce.string().max(255).optional(),
+  destination: z.coerce.string().max(255).optional(),
+  purposeOfTransportation: z.coerce.string().optional(),
+  cargoList: z.coerce.string().optional(),
+  placeValidity: z.coerce.string().optional(),
+  transportationPassengers: z.array(
+    z.object({
+      studentNumber: z.coerce.string(),
+    }),
+  ),
+
+  isNonCorporateTransaction: z.coerce.boolean(),
+  traderName: z.coerce.string().max(255).optional(),
+  traderAccountNumber: z.coerce.string().max(255).optional(),
+  wasteExplanation: z.coerce.string().optional(),
+
+  isFoodExpense: z.coerce.boolean(),
+  isLaborContract: z.coerce.boolean(),
+  isExternalEventParticipationFee: z.coerce.boolean(),
+  isPublication: z.coerce.boolean(),
+  isProfitMakingActivity: z.coerce.boolean(),
+  isJointExpense: z.coerce.boolean(),
+  isEtcExpense: z.coerce.boolean(),
+
+  foodExpenseExplanation: z.coerce.string().optional(),
+  laborContractExplanation: z.coerce.string().optional(),
+  externalEventParticipationFeeExplanation: z.coerce.string().optional(),
+  publicationExplanation: z.coerce.string().optional(),
+  profitMakingActivityExplanation: z.coerce.string().optional(),
+  jointExpenseExplanation: z.coerce.string().optional(),
+  etcExpenseExplanation: z.coerce.string().optional(),
+
+  foodExpenseFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  laborContractFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  externalEventParticipationFeeFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  publicationFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  profitMakingActivityFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  jointExpenseFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+  etcExpenseFiles: z.array(
+    z.object({
+      uid: z.coerce.string().max(255),
+    }),
+  ),
+});
+
+const requestBody = requestPreBody.superRefine((data, ctx) => {
+  if (data.purposeId === undefined) {
+    if (
+      !data.clubSuppliesName ||
+      !data.clubSuppliesEvidenceEnumId ||
+      !data.clubSuppliesClassEnumId ||
+      !data.numberOfClubSupplies ||
+      !data.priceOfClubSupplies
+    ) {
+      ctx.addIssue({
+        code: z.ZodIssueCode.custom,
+        message: "clubSupplies is required",
+      });
+    }
+
+    if (data.clubSuppliesClassEnumId === 4) {
```

Comment:
💊 EnumId는 Enum을 number로 캐스팅한 값이랑 비교하면 의미가 더 정확해질 거 같아요
```suggestion
    if (data.clubSuppliesClassEnumId === number(clubSuppliesClassEnumId.Software)) {
```